src/litdata/streaming/config.py

```diff
@@ -83,7 +83,14 @@ def __init__(
         self._compressor_name = self._config["compression"]
         self._compressor: Optional[Compressor] = None
 
-        if self._compressor_name in _COMPRESSORS:
+        if self._compressor_name:
+            if len(_COMPRESSORS) == 0:
+                raise ValueError("No compresion algorithms are installed.")
+
+            if self._compressor_name not in _COMPRESSORS:
+                raise ValueError(
```

Comment:
Btw, can we try to install the required compression algorithm rather than raising ValueError?

Something like, 
```python
print("No compression algorithms are installed")
print("Installing ....")
```

what do you think?

Reply:
Thank you for the review and suggestion @deependujha. That could indeed be a helpful approach. 

However, I’m wondering if it is better to leave the installation process to the user, possibly providing them with clear instructions or recommendations on how to proceed with the installation.